saleor/graphql/stock/types.py

```diff
@@ -0,0 +1,31 @@
+import graphene
+from django.conf import settings
+
+from ...stock import models
+from ..core.connection import CountableDjangoObjectType
+from ..decorators import permission_required
+
+
+class StockInput(graphene.InputObjectType):
+    product_variant = graphene.ID(
+        required=True, description="Product variant assiociated with stock."
+    )
+    warehouse = graphene.ID(
+        required=True, description="Warehouse in which stock is lockated."
+    )
+    quantity = graphene.Int(description="Quantity of items available for sell.")
+    quantity_allocated = graphene.Int()
```

Comment:
Don't you need default value here? like `0`?

Reply:
Default value is set in DB (to 0)

Reply:
The description is missing here as well.